assets/app/views/browse/routes.html

```diff
@@ -29,11 +33,21 @@
               </div>
             </td>
             <td data-title="Routes to">
```

Comment:
since you capitalized the other header you need to change this one as well (this is the mobile header)